### ISSUE TITLE
NFInst.instExpressions fails because of scoping issue in Buildings 7.0.1

### ISSUE THREAD

**Issue by casella** [OpenModelica/OpenModelica#7837]
Please check [Buildings.ThermalZones.Detailed.Constructions.Examples.ExteriorWallTwoWindows](https://libraries.openmodelica.org/branches/master/Buildings_7.0.0/files/Buildings_7.0.0_Buildings.ThermalZones.Detailed.Constructions.Examples.ExteriorWallTwoWindows.err). NFInst.instExpressions fails with
```
[Buildings 7.0.1/ThermalZones/Detailed/Constructions/Examples/ExteriorWallTwoWindows.mo:118:5-118:11:writable]
Error: Variable AGla not found in scope ExteriorWallTwoWindows.
```

**Comment by perost** [comment]
This was fixed in the library in lbl-srg/modelica-buildings#2558

**Comment by casella** [comment]
@perost, I added a comment on the LBL ticket regarding porting the fix to 7.0.1 and including it in our tests. I'd close this ticket when this is resolved and we get successful output on the [CI report](https://libraries.openmodelica.org/branches/master/Buildings_7.0.0/files/Buildings_7.0.0_Buildings.ThermalZones.Detailed.Constructions.Examples.ExteriorWallTwoWindows.err).

**Comment by perost** [comment]
Fixed, [see report](https://libraries.openmodelica.org/branches/master/Buildings_maint.7.0.x/files/Buildings_maint.7.0.x_Buildings.ThermalZones.Detailed.Constructions.Examples.ExteriorWallTwoWindows.err).
